packages/core/test/ovm/Finder.js

```diff
@@ -0,0 +1,58 @@
+/* External Imports */
+const { web3, artifacts } = require("hardhat");
+const { didContractThrow } = require("@uma/common");
+const { utf8ToHex, hexToUtf8, toChecksumAddress, randomHex, padRight } = web3.utils;
+const Finder = artifacts.readArtifactSync("Finder");
```

Comment:
this is new syntax. why did you import like this? is this the workaround for the OVM compiled contracts?

Reply:
Yes because `getAbi` doesn't know to look in the `artifacts-ovm` directory. More generally, `getAbi` looks in the `build/contracts` dir which is where Truffle places compiled contracts. 

Since there is not Optimism plugin for Truffle, you'll notice that this script is NOT a truffle script (i.e. it does not start with `contract("Finder", function(accounts){...})`, so its run by hardhat. Therefore there is an injected `hardhat runtime environment` which makes it trivial to read artifacts like so.

I actually think if we're going to be running more tests with hardhat, we should move to use syntax like this.

Reply:
@chrismaree I discussed this with @nicholaspai offline. I'm working on standardizing the syntax for a "hardhat test" (i.e. one that doesn't require the truffle plugin). In that vein, I will be sending out a PR with these practices/plugins written out, so we can align around them and @nicholaspai can use them here.

Reply:
See https://github.com/UMAprotocol/protocol/pull/3180.